generators/setup-workspace/index.js

```diff
@@ -361,7 +365,8 @@ class Generator extends Base {
       })
       .then((repos) => Promise.all(repos.map((r) => this._cloneRepo(r.repo, r.branch))))
       .then(this._yo.bind(this, 'workspace', {
-        defaultApp: findDefaultApp()
+        defaultApp: findDefaultApp(),
+        noLogging: true// do not show logs from yo phovea:workspace
```

Comment:
```suggestion
        skipNextStepsLog: true // skip "next steps" logs from yo phovea:workspace
```

See related changes in workspace generator.